src/Mailbox.php

```diff
@@ -134,6 +135,24 @@ public function getStatus(int $flags = null): \stdClass
         return $return;
     }
 
+    /**
+     * Get Mailbox quota.
+     *
+     * @param string $root
+     *
+     * @return array
+     */
+    public function getQuota(string $root = 'INBOX'): array
+    {
+        $return = \imap_get_quotaroot($this->resource->getStream(), $root);
```

Comment:
Most of the times this feature refers to the entire account, not to a specific mailbox, as also the [PHP doc says](https://www.php.net/manual/en/function.imap-get-quotaroot.php#refsect1-function.imap-get-quotaroot-parameters):

>  `$quota_root` should normally be in the form of which mailbox (i.e. INBOX).

So the entire method should be put into the `Connection` class

Reply:
I also though about putting it into the Connection class. Therefore I'm fine with this.